src/languageServerInstaller.ts

```diff
@@ -52,11 +52,12 @@ interface Release {
 export class LanguageServerInstaller {
 	public async install(directory: string): Promise<void> {
 		const { version: extensionVersion } = require('../package.json');
-		const lspCmd = `${directory}/terraform-ls --version`;
+		const lsVersionCmd = `${directory}/terraform-ls --version`;
 		const userAgent = `Terraform-VSCode/${extensionVersion} VSCode/${vscode.version}`;
 		let isInstalled = true;
 		try {
-			var { stderr: installedVersion } = await exec(lspCmd);
+			var { stdout: stdout, stderr: stderr } = await exec(lsVersionCmd);
```

Comment:
object destructuring only need the `name: othername` if you want to pull out a key, but rename it to `othername` as I did before. if you keep the keyname intact you can simplify to
```
var { stdout, stderr } = await exec(lsVersionCmd);
```

Reply:
I see, let me fix that 😄